docs/run/instances.mdx

```diff
@@ -24,6 +24,16 @@ To see the instances to which you have access, look at the bottom of your [Accou
 
 You can view a list of your instances on your [account settings page](https://quantum-computing.ibm.com/account), or you can use [the `instances()` method](../api/qiskit-ibm-runtime/qiskit_ibm_runtime.QiskitRuntimeService#instances).
 
+## Switch instances
+
+If you have access to run on multiple instances, the [IBM Quantum interface](https://quantum-computing.ibm.com/) menu bar contains a dropdown that lets you switch between instances.  The IBM Quantum Platform dashboard, Compute resources, and Jobs pages display information such as usage metrics, jobs, and systems based on the selected instance.  
+
+<Admonition type="note">
+ The instance switcher does not appear in the Administration application.
+ </Admonition>
+
+If you switch to a different instance, it is remembered the next time you log on and, assuming that it's still a valid instance, information pertaining to that instance is displayed.  By default, the first (alphabetically) premium instance you have access to is used.  If you do not have any premium instances, the first (alphabetically) open instance is shown. 
```

Comment:
I wonder if we can move the word "alphabetically" before the word "first". Otherwise it could read as "alphabetically premium"...

Reply:
After discussion, we're going with a different edit.